config/201-clusterrole.yaml

```diff
@@ -35,6 +35,14 @@ rules:
   - patch
   - delete
 
+# Sources finalizer
+- apiGroups:
+  - sources.eventing.knative.dev
+  resources:
+  - '*/finalizers'
```

Comment:
there is not another example of * in all of knative, I think we should be explicit.